docs/book/introduction.md

```diff
@@ -4,27 +4,178 @@ description: Welcome to ZenML!
 
 # ⭐ Introduction
 
-🤹 Are you an ML engineer or data scientist shipping models to production while juggling a plethora of tools?
+ZenML is an extensible, open-source MLOps framework for creating portable, 
+production-ready machine learning pipelines.
+By decoupling infrastructure from code, ZenML gives all developers in your 
+organization more freedom and independence in how they approach their work.
```

Comment:
"gives freedom and independence" feels a little vague. This is an important second sentence where we give the 'why' / value prop for ZenML so it should be super tight. (Sorry no immediate suggestions come to mind on how to rephrase, but I think it needs work).

Reply:
Okay let's see if you like the new version better